pkg/asset/handler/presign_upload.go

```diff
@@ -140,18 +139,13 @@ func (h *PresignUploadHandler) ServeHTTP(w http.ResponseWriter, r *http.Request)
 	handler.WriteResponse(w, response)
 }
 
-func (h *PresignUploadHandler) ParsePresignUploadRequest(r io.Reader, p interface{}) error {
-	return h.Validator.ParseReader("#PresignUploadRequest", r, p)
-}
-
 func (h *PresignUploadHandler) Handle(w http.ResponseWriter, r *http.Request) (result interface{}, err error) {
 	// Parse request
 	var payload cloudstorage.PresignUploadRequest
-	err = handler.ParseJSONBody(r, w, h.ParsePresignUploadRequest, &payload)
+	err = handler.BindJSONBody(r, w, h.Validator, "#PresignUploadRequest", &payload)
```

Comment:
I think we can remove ParseJSONBody since it is replaced by the better BindJSONBody?

Reply:
`ParseJSONBody` is used by `DecodeJSONBody`, which is used by handlers accepting empty JSON body. If we'd like to remove them, then these handlers need to validate against the `emptyResponseSchema` schema.